db/deprecated/utils.py

```diff
@@ -27,11 +28,26 @@ def execute_statement(engine, statement, connection_to_use=None):
             raise e
 
 
+def stringify_json_cols(query):
+    col_list = []
+    for col in query.c:
+        if isinstance(col.type, (JSONB, JSON)):
+            col_list.append(sqlalchemy.cast(col, TEXT))
+        elif isinstance(col.type, ARRAY) and isinstance(col.type.item_type, (JSONB, JSON)):
+            new_col = sqlalchemy.literal_column(
+                f"(SELECT jsonb_agg(x::text) FROM unnest({str(col)}) AS x)"
+            ).label(col.name)
+            col_list.append(new_col)
+        else:
+            col_list.append(col)
+    return col_list
```

Comment:
Letting all the `isinstance` calls pass since this is in the deprecated.

This should use `array_agg` instead of `jsonb_agg`.